docs_src/subcommands/name_help/tutorial007.py

```diff
@@ -21,6 +21,7 @@ def new_users():
 app.add_typer(users_app, callback=new_users)
 
 
+# TODO: do we want this to work?
 @users_app.callback("call-users", help="Help from callback for users.")
```

Comment:
@tiangolo I think we might want to remove support for this too 😊

Reply:
Yep, I think so.